unlock-protocol-com/blog/subgraph-v2.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Subgraph upgrade
```

Comment:
please use "Title Case" for titles and change to:
**Unlock Protocol Subgraph Upgrade with The Graph**